### PR TITLE
Fixed type of the "host" parameter in asyncio getaddrinfo()

### PR DESCRIPTION
**Pull request by agronholm**
The types should match `socket.getaddrinfo()`.

### CODE REVIEW

File: stdlib/asyncio/events.pyi [hunk 1]
```diff
@@ -211,7 +211,14 @@ class AbstractEventLoop:
     # Network I/O methods returning Futures.
     @abstractmethod
     async def getaddrinfo(
-        self, host: str | None, port: str | int | None, *, family: int = ..., type: int = ..., proto: int = ..., flags: int = ...
+        self,
+        host: bytes | str | None,
```
Comment (AlexWaygood) [inline stdlib/asyncio/events.pyi:215]: Any reason not to include `bytearray` in the union as well, like we do in `socket.getaddrinfo()`?

```suggestion
        host: bytearray | bytes | str | None,
```

Comment (srittau) [reply stdlib/asyncio/events.pyi:215]: `bytearray` is [redundant with `bytes` in argument types](https://docs.python.org/3/library/typing.html#typing.ByteString) and shouldn't be included.

Comment (AlexWaygood) [reply stdlib/asyncio/events.pyi:215]: > `bytearray` is [redundant with `bytes` in argument types](https://docs.python.org/3/library/typing.html#typing.ByteString) and shouldn't be included.

Okay — no action to be taken here then; we should just change `socket.getaddrinfo`

**Review by AlexWaygood — APPROVED** [review]

**Comment by agronholm** [comment]
Thanks!

**Pull request merged**